### PR TITLE
Fix repropose

### PR DESCRIPTION
**Pull request by jreidinger**

### CODE REVIEW

File: package/yast2-bootloader.changes [hunk 1]
```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Wed Jun  3 11:44:23 UTC 2015 - jreidinger@suse.com
+
+- Fix ignoring bootloader settings after changing it in proposal
```
Comment (imobach) [inline package/yast2-bootloader.changes]: np: 'changing them' instead of 'changing it'.

**Comment by lslezak** [comment]
LGTM

**Comment by imobach** [comment]
Just a minor grammar issue. Otherwise, LGTM.

File: package/yast2-bootloader.changes (hunk 1 above)
Comment (jreidinger) [inline package/yast2-bootloader.changes]: thanks, fixed

**Comment by jreidinger** [comment]
thanks, lets wait for travis now

**Pull request merged**